components/table/table-wrapper.js

```diff
@@ -362,7 +366,8 @@ export class TableWrapper extends RtlMixin(LitElement) {
 			this._tableIntersectionObserver.observe(table);
 		}
 
-		requestAnimationFrame(() => this._applyClassNames(table));
+		this._applyClassNames(table);
```

Comment:
I moved the `requestAnimationFrame` inside `_applyClassNames` so this isn't needed.